.github/workflows/release.yml

```diff
@@ -310,140 +291,139 @@ jobs:
-      run: |
-        git config --global user.email "richard@rghsoftware.co.uk"
-        git config --global user.name "GitHub Actions bot"
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install Jupyter dependency
-      run: |
-        pip install jupyter ipynb-py-convert
-    - name: Update jupyter notebooks
-      run: |
-        pushd PyAutoBuild
-        export PYTHONPATH=$PYTHONPATH:../
-        python3 "${{ matrix.workspace.name }}/generate.py"
-        popd
-        pushd "${{ matrix.workspace.path}}"
-        git add *.ipynb
-        git commit -m "Updated notebooks"
-    - name: Release
-      run: |
-        cd "${{ matrix.workspace.path }}"
-        git checkout release
-        git merge master
-        VERSION="${{ needs.version_number.outputs.version_number }}"
-        git tag -m "Release $VERSION" -a "$VERSION"
-        git push
-        git push --tags
\ No newline at end of file
+#  release:
+#    runs-on: ubuntu-latest
+#    env:
+#      TWINE_REPOSITORY: pypi
+#      TWINE_USERNAME: __token__
+#      TWINE_PASSWORD: ${{ secrets.PYPI }}
+#    needs:
+#      - run_scripts
+#      - run_notebooks
+#      - release_test_pypi
+#      - version_number
+#    strategy:
+#      matrix:
+#        project:
+#        - repository: rhayes777/PyAutoConf
+#          ref: master
+#          path: PyAutoConf
+#          pat: PAT_RICH
+#        - repository: rhayes777/PyAutoFit
+#          ref: master
+#          path: PyAutoFit
+#          pat: PAT_RICH
+#        - repository: Jammy2211/PyAutoArray
+#          ref: master
+#          path: PyAutoArray
+#          pat: PAT_JAMES
+#        - repository: Jammy2211/PyAutoGalaxy
+#          ref: master
+#          path: PyAutoGalaxy
+#          pat: PAT_JAMES
+#        - repository: Jammy2211/PyAutoLens
+#          ref: master
+#          path: PyAutoLens
+#          pat: PAT_JAMES
+#    steps:
+#    - uses: actions/checkout@v2
+#      with:
+#          path: PyAutoBuild
+#    - name: Checkout
+#      run: |
+#        git clone -b "${{ matrix.project.ref }}" "https://$PAT@github.com/${{ matrix.project.repository }}.git" "${{ matrix.project.path }}"
+#    - name: Set up Python ${{ matrix.python-version }}
+#      uses: actions/setup-python@v2
+#      with:
+#        python-version: ${{ matrix.python-version }}
+#    - name: Configure Git
+#      run: |
+#        git config --global user.email "richard@rghsoftware.co.uk"
+#        git config --global user.name "GitHub Actions bot"
+#    - name: Update version
+#      run: |
+#        pushd "${{ matrix.project.path }}"
+#        VERSION="${{ needs.version_number.outputs.version_number }}"
+#        sed -i "s/__version__ = [\.\"\'0-9]*/__version__ = \"$VERSION\"/g" */__init__.py
+#        git commit "-am 'Updated version in __init__ to $VERSION"
+#    - name: Tag
+#      run: |
+#        pushd "${{ matrix.project.path }}"
```

Comment:
I assume you want to coment this back in?

Reply:
Yep, once I've tested everything end-to-end.